transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/auth/BasicLoginModuleTestIT.java

```diff
@@ -37,9 +37,9 @@
                     .accept("ws://localhost:8001/echoAuth")
                     .type("echo")
                     .realmName("demo")
-                        .authorization()
-                        .requireRole("USER")
-                    .done()
+//                        .authorization()
```

Comment:
We should remove these comments, or duplicate the test and have one with the comments removed and one with them there